rest/rest-resources/pom.xml

```diff
@@ -220,10 +215,9 @@
                 <configuration>
                     <instructions>
                         <Import-Package>
-                            org.apache.karaf.jaas.config,
-                            org.eclipse.jetty.jaas,
                             *
                         </Import-Package>
+                        <Export-Package>org.apache.brooklyn.*</Export-Package>
```

Comment:
Is this belt-and-braces, or does it contain public things in packages other than `org.apache.brooklyn.*`?

Reply:
not sure i understand the question; this bundle provides 5 or so packages, and most of them are things other projects might want to use, so we export those 5 packages.  does that help?